easybuild/easyconfigs/f/FigureGen/FigureGen-51-foss-2019a.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CmdCp'
+
+name = 'FigureGen'
+version = '51'
+
+homepage = 'https://ccht.ccee.ncsu.edu/figuregen-v-49/'
+description = """  FigureGen is a Fortran program that creates images for ADCIRC files.
+It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output 
+files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors. 
+Using FigureGen, you can go directly from the ADCIRC input and output files to a 
+presentation-quality figure, for one or multiple time snaps.  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

If `FigureGen` has to be compiled with `mpi`, then it should be enabled in the toolchain (see below in `cmds_map` for an explanation)
```suggestion
toolchain = {'name': 'foss', 'version': '2019a'}
toolchainopts = {'usempi': True}
```